spec/swagger_helper.rb

```diff
@@ -33,6 +33,21 @@
       #   }
       # },
       definitions: {
+        payment: {
+          type: :object,
+          properties: {
+            agency_id: { type: :string, example: '3fa57706-f5bb-4d40-9350-85871f698d52' },
+            amount_cents: { type: :integer, example: '19999' },
```

Comment:
Same question as for above in the spec params; should this really take in `amount_cents` or just `amount`?

Reply:
..._cents